### ISSUE TITLE
[Feature]: SD Forge Attention Couple better alternative to regional prompter with gui

### ISSUE THREAD

**Issue by Bercraft** [vladmandic/automatic#3122]
### Feature description

This is an Extension for the which allows you to target conditioning at different regions. No more color bleeds or mixed features!

link to extension : https://github.com/Haoming02/sd-forge-couple#advanced-mapping

**Comment by Aptronymist** [comment]
> ### Feature description
> This is an Extension for the which allows you to target conditioning at different regions. No more color bleeds or mixed features!
> 
> link to extension : https://github.com/Haoming02/sd-forge-couple#advanced-mapping

Sadly this is specifically created for Forge, so I seriously doubt we could use it entirely, but perhaps the UI part of it?
I'll leave that for vlad to check out, if so that would be an improvement over the basic version we have by far, which I agree does need a UI of some sort.

**Comment by Bercraft** [comment]
I opened an issue to the creator git, he said its a backend compatibility issue.
Yes i think with just the ui the reginal prompter would be really usefull and capable.
Thank you for your efforts.

**Comment by Aptronymist** [comment]
> I opened an issue to the creator git, he said its a backend compatibility issue. Yes i think with just the ui the reginal prompter would be really usefull and capable. Thank you for your efforts.

Oh yes, many many things won't work with us because they're looking for a1111 specific code/features that we do with diffusers or vlad has rewritten, especially things that involve latent manipulations. But that doesn't mean we can't take some bits here and there or inspiration for doing it better ourselves.

**Comment by Bercraft** [comment]
Yes i have fully understood, what now i close the request or left it open as reminder?

**Comment by vladmandic** [comment]
leave it open

**Comment by Bercraft** [comment]
I do not want to trash this request, but using the latest dev in particular the mixture tiling script, it does an awesome job but the lack of a gui is sometimes frustrating.

I also read the git hub and th gui for it is basically the regional prompter part of this extension [multidiffusion-upscaler-for-automatic1111](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111), as said on the git

"If you prefer a more user friendly graphical interface to use this algorithm, I recommend trying the [Tiled Diffusion & VAE](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111) plugin developed by pkuliyi2015 for [AUTOMATIC1111's stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui)."

thank you for your work.

**Comment by lbeltrame** [comment]
The major difference in the extension linked compared to what's in SD.Next currently is support for SDXL.

**Comment by Bercraft** [comment]
> The major difference in the extension linked compared to what's in SD.Next currently is support for SDXL.

Sorry i do not want to contraddict, but the difference in not only what you mentioned, This request was opened to  get a gui for all the great script that are already included with sdnext. The extension was linked just because in sdnext was added a feature called "mixture tiling" and its based on this repo "https://github.com/albarji/mixture-of-diffusers" and the author adviced that a gui is already present inside this "regional prompter part of this extension [multidiffusion-upscaler-for-automatic1111](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111), as said on the git".
i did not change the request name just because i am waiting for vlad or others to ask it etc. just because i do not want to cause trouble.

**Comment by lbeltrame** [comment]
Oh, no problem at all. I'm aware you mentioned the GUI, but I also wanted to point out that even at the basic, non GUI functionality there's a difference (and as far as I remember, even mixture tiling in SD.Next is SD 1.5 only).
